config/v1/types_scheduling.go

```diff
@@ -28,20 +28,16 @@ type Scheduler struct {
 }
 
 type SchedulerSpec struct {
-	// DEPRECATED: the scheduler Policy API has been deprecated and will be removed in a future release.
-	// policy is a reference to a ConfigMap containing scheduler policy which has
-	// user specified predicates and priorities. If this ConfigMap is not available
-	// scheduler will default to use DefaultAlgorithmProvider.
-	// The namespace for this configmap is openshift-config.
-	// +optional
-	Policy ConfigMapNameReference `json:"policy,omitempty"`
 	// profile sets which scheduling profile should be set in order to configure scheduling
 	// decisions for new pods.
 	//
 	// Valid values are "LowNodeUtilization", "HighNodeUtilization", "NoScoring"
 	// Defaults to "LowNodeUtilization"
 	// +optional
 	Profile SchedulerProfile `json:"profile,omitempty"`
+	// ProfileCustomizations contains various parameters for modifying the default behavior of certain profiles.
+	// Experimental implementation for profile customization.
+	ProfileCustomizations *ProfileCustomizations `json:"profileCustomizations,omitempty"`
```

Comment:
What's the motivation for this field to be a pointer, does it need to be `omitempty`?

Reply:
Just aligning with the descheduler customization API. Though, the customization configuration can always be present with default value for each field.

Reply:
Lets drop the pointer and omitempty, it will improve the discoverability for an API such as this

Reply:
Done